src/style.css

```diff
@@ -1 +1,96 @@
-/* Styles go here */
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+li {
+  list-style: none;
+}
+
+* {
+  all: unset;
+  display: revert;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: grey;
+  margin: 5px;
+}
+
+.conteiner {
+  display: flex;
+  background-color: white;
+  padding: 0 49px;
+  padding-right: 50px;
+  flex-direction: row;
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
+
+.navigation {
+  display: flex;
+  position: relative;
+  flex-wrap: wrap;
+  height: auto;
+  padding-top: 19px;
+}
+
+.nav__img {
+  margin-top: 10px;
+  margin-bottom: 12px;
+}
+
+.nav__link {
+  position: relative;
+  font-size: 12px;
+  font-style: normal;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: black;
+  margin-left: 20px;
+  padding-top: 23px;
+  padding-bottom: 18px;
+  height: 60px;
+}
+
+a:hover {
+  display: inline;
+  color: #00acdc;
+  cursor: pointer;
+}
+
+.nav__link:active::after {
+  content: "";
+  position: absolute;
+  top: 56px;
+  left: 0;
+  background-color: #00acdc;
+  width: 100%;
+  height: 4px;
+  border-radius: 2px;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+  margin-left: 0;
+}
+
+.is-active::after {
```

Comment:
.nav__link:active::after and .is-active::after are the same. Don't duplicate yourself

Reply:
this styles make difrent action